src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color[] colors = Color.values();
+
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(colors.length);
```

Comment:
Do we really need to create a new instance of `Random` every time when this method is called?
It is better to create a new `Random` field as class property.